askomics/libaskomics/rdfdb/QueryLauncher.py

```diff
@@ -84,46 +84,46 @@ def format_results_csv(self, table):
             fp.write(table)
         return "/static/results/"+os.path.basename(fp.name)
 
+    # TODO see if we can make a rollback in case of malformed data
```

Comment:
A Virtuoso développer blog post about transactions in RDF : http://www.openlinksw.com/dataspace/doc/oerling/weblog/Orri%20Erling%27s%20Blog/1689

This require a *DBC connection, which may note be portable across triplestores.